tests/test_sr.py

```diff
@@ -1872,7 +1871,7 @@ def test_construction_with_frame_reference(self):
         )
         assert (
```

Comment:
Change made so that unit tests pass - was there a change in how frame numbers were managed?

Reply:
Indeed, see #162 and its resolution in #163

Reply:
Basically it was a breaking change to the way pydicom deals with multivalue elements that have a single value, introduced in pydicom 2.3.0, it broke a bunch of stuff. This fix is fine